src/core/less/base.less

```diff
@@ -28,8 +28,8 @@
     display:none!important;
 }
 
-.accessibility.no-touchevents {
-    .focused, button:focus, select:focus, textarea:focus, a:focus, input:focus + label {
+.accessibility {
+    *:focus {
```

Comment:
Would you mind ditching the asterisk? It's implied and we don't seem to use it elsewhere.